app/src/main/res/values/strings.xml

```diff
@@ -542,8 +542,8 @@
   <string name="appc_home_bundle_iab">Use Your AppCoins</string>
   <string name="appc_home_bundle_poa">Earn AppCoins Credits</string>
   <string name="appc_title_about_appcoins">About AppCoins</string>
-  <string name="appc_message_appview_appcoins_reward">Try this app and get %s</string>
-  <string name="appc_message_appview_appcoins_reward_with_value">Try this app for 2 minutes and get %1$s %2$s</string>
+  <string name="appc_message_appview_appcoins_reward">Try this app for 2 minutes and get\n%s</string>
```

Comment:
same changes in the strings file as the other pull request. Did Carlos review this?